package.json

```diff
@@ -186,12 +195,16 @@
     "jquery.transition": "dzucconi/jquery.transition",
     "jsdom": "^11.0.0",
     "jsdom-global": "^3.0.2",
+    "json-loader": "^0.5.7",
     "konami-js": "0.0.2",
     "mocha": "^3.1.2",
     "nightmare": "^2.10.0",
     "openseadragon": "^2.2.1",
+    "progress-bar-webpack-plugin": "^1.10.0",
+    "pug-loader": "damassi/pug-loader#02921c5",
```

Comment:
Had to fork a fork to make this work

Reply:
For small changes I nowadays like to use https://github.com/ds300/patch-package rather than maintaining a fork, unless it’s generic enough and you’re planning to send it upstream?

Reply:
Interesting, will update. `pug-loader` is basically abandoned at this point, so don't expect this to be accepted. Also had to modify some of the expected behavior to make it more like `jadify`.

Reply:
This is much nicer than 

<img width="526" alt="screen shot 2017-12-26 at 3 38 11 pm" src="https://user-images.githubusercontent.com/236943/34366603-d82089ca-ea52-11e7-8a57-dd78d26f20f3.png">